### ISSUE TITLE
HW3 question on author-input in Write tab of the Article write page 

### ISSUE THREAD

**Issue by Zagoshipda** [swsnu/swppfall2021#76]
![hw3question](https://user-images.githubusercontent.com/34166218/136141558-5c298ad1-0940-4a75-a421-d0125de693a6.PNG)

I have a question about the above spec. While ___article-title___ & ___article-content___ fields both exist in write tab and preview tab,

- in the write tab there is NO ___article-author-input___ field
- but in the preview tab there is ___article-author___ field

I think we should type the author-id so the author input is omitted. Is it right or is there something I am missing?
thanks :)

**Comment by Zagoshipda** [comment]
@kdh0102 Then I think the author must be automatically same as the current logged-in user, right? I'm asking because I just thought that the user and author can be different. I got confused betweeen those two

**Comment by kdh0102** [comment]
Yes, the author is the logged-in user if you are creating an article.
Note that the author may not be the logged-in user in other articles. For instance, `Alan Turing` in our db.json can be the author.
